docs/source-2.0/languages/typescript/quickstart.rst

```diff
@@ -0,0 +1,315 @@
+=====================
+TypeScript Quickstart
+=====================
+
+This guide introduces `Smithy TypeScript <https://github.com/smithy-lang/smithy-typescript>`_ with a simple working example of a
+generated server and client.
+
+For this example, imagine that you are the proud owner of a coffee shop.
+Your API allows your customers to order some *java* from their TypeScript applications.
```

Comment:
```suggestion
Your API allows your customers to order coffee from their TypeScript applications.
```
This pun was intended because of smithy-java. Not needed here.